prov-ui/src/pages/UpdateModelRunTool.tsx

```diff
@@ -156,6 +163,18 @@ export const UpdateModelRunTool = () => {
   const granted = !!auth.granted;
   const showUpdateForm = granted && canWriteToRecord && !showResult;
 
+  /* Use effect that runs once on component load to check whether 
+     a model run id was provided to the component. If model run id 
+     is provided, the state is set to the respective provided ID,
+     AND the model run id in the parent component is reset.
+  */
+  useEffect(() => {
+    if(props && props.modelRunId){
+      setModelRunId(props.modelRunId)
+      props.reset()
+  }
+  }, [])
+
```

Comment:
just pass through the parent prop instead of syncing it here, but this is okay for now